### DISCUSSION TITLE
GeoData datatype [Apr 23]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5564] [Discord Threads]
> 🙋‍♂️ **ibarot**  
## Hi, I see in the docs there is a datatype for GeoData, but I cannot find it in my project.  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1088754208062328883/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### GeoData & MapView are in still in beta. We would be happy to discuss your use case.  
#  
> 🙋‍♂️ **ibarot**  
#### Oh, so that is why its not showing up. I figured something is wrong with my configuration. My use case is storing my suppliers address along with geo data so I can calculate distance and freight somehow. I was just exploring how to go about that.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1088754208267829268)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
